### DISCUSSION TITLE
How to hide adjusting overlay when turning knobs?

### DISCUSSION THREAD

**Discussion by katerlouis** [rsjaffe/MIDI2LR#686] [Q&A] [answered]
Probably already been covered, but couldn't find anything to "overlay". 

I'd like to get rid of this overlay when turning the knobs. Is that possible?

<img width="299" alt="image" src="https://user-images.githubusercontent.com/1983382/185419052-4ac6cb4b-70b2-428a-b1e1-35dbaefa38dd.png">

This does not happen when hovering a slider with the mouse and pressing UP or DOWN arrow.

**Comment by PitToad** [comment, marked as answer]
In Windows, at least;

File, PlugIn Extras, General Options
'Other' tab
Un-tick 'Show Status And Activity'

![overlay](https://user-images.githubusercontent.com/111119318/185458548-d6be5bd4-fa10-40ff-9d66-46e981121e71.jpg)

![overlay2](https://user-images.githubusercontent.com/111119318/185458571-66111a02-032a-4d6e-868b-845626b8f2a7.jpg)

**Reply by katerlouis** [reply]
It's pretty much the same for macOS. Thanks a lot for this detailed explanation! Especially thanks for the extra effort in highlighting the screenshots. 

I didn't know there were options like this! Wondering why they are not in the main window or at least linked to? And now I don't have to look up how the keyboard shortcuts are supposed to work :P

**Comment by katerlouis** [comment]
I'm also wondering why this setting is switched on by default? I guess only power users and go to the length of setting up such a workflow. I'd argue that most of these power users are annoyed by having their adjustment overlayed.

**Comment by PitToad** [comment]
I don't know what constitutes a power user but, on average, I edit pics in batches of around 1800 at a time and love having the overlay - Couldn't tell you why as I don't refer to it, but I've got used to it being there :-)

When I moved on from LR6 I had to find out how to enable it, so 'on by default' is newish as it was there on 6 but not, originally, in the Cloud version.

**Comment by rsjaffe** [comment]
The info is in the wiki: specifically https://github.com/rsjaffe/MIDI2LR/wiki/Plugin-Options-Dialog#other-settings . I made it default based on the idea of "least surprising": this way, the application gives feedback showing what it's doing. It is configurable.
